python/cac_tripplanner/cac_tripplanner/settings.py

```diff
@@ -94,3 +94,10 @@
 # https://docs.djangoproject.com/en/1.7/howto/static-files/
 
 STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'cac_tripplanner', 'static')
+BUILD_DIR = '/opt/app/src'
```

Comment:
It might be good to put this in the secrets dict above so we can modify with group_vars later.